ev3dev2/motor.py

```diff
@@ -2670,66 +2761,26 @@ def _odometry_monitor():
                 if sleep_time:
                     time.sleep(sleep_time)
 
-            self.odometry_thread_id = None
+        _thread.start_new_thread(_odometry_monitor, ())
 
-        self.odometry_thread_run = True
-        self.odometry_thread_id = _thread.start_new_thread(_odometry_monitor, ())
```

Comment:
The removal of `odometry_thread_id` could be breaking.  I modified this because `_thread.start_new_thread` in micropython returns None so the behavior was inconsistent.  I could put this back the way it was but just not use `odometry_thread_id` anywhere.